app/serializers/current_user_serializer.rb

```diff
@@ -309,20 +308,12 @@ def include_has_topic_draft?
     Draft.has_topic_draft(object)
   end
 
-  def experimental_sidebar_enabled
-    object.user_option.enable_experimental_sidebar
```

Comment:
The option is no longer used and I'll follow up with a PR to drop the column in the future.